versions.yml

```diff
@@ -149,17 +151,23 @@ modGroups:
       desc: Rewrites light engine to reduces chunk load stuttering
       url: https://cdn.modrinth.com/data/H8CaAYZC/versions/1.1.1+1.19/starlight-1.1.1%2Bfabric.ae22326.jar
 
+    - name: Lithium
+      license: LGPL-3
+      homepage: https://modrinth.com/mod/lithium
+      desc: General-purpose performance enhancements
+      url: https://github.com/CaffeineMC/lithium-fabric/releases/download/mc1.19-0.8.0/lithium-fabric-mc1.19-0.8.0.jar
+
     - name: LazyDFU
       license: MIT
       homepage: https://modrinth.com/mod/lazydfu
       desc: Makes the game start more quickly
       url: https://github.com/astei/lazydfu/releases/download/v0.1.3/lazydfu-0.1.3.jar
 
-    - name: Lithium
-      license: LGPL-3
-      homepage: https://modrinth.com/mod/lithium
-      desc: General-purpose performance enhancements
-      url: https://github.com/CaffeineMC/lithium-fabric/releases/download/mc1.19-0.8.0/lithium-fabric-mc1.19-0.8.0.jar
+    - name: Smooth Boot
+      license: MIT
+      desc: Improves boot CPU usage and times
+      homepage: https://modrinth.com/mod/smoothboot-fabric
+      url: https://cdn.modrinth.com/data/FWumhS4T/versions/1.19-1.7.1/smoothboot-fabric-1.19-1.7.1.jar
 
```

Comment:
This really better? afaik lithium is made specifically with sodium and that group in mind